src/fe-text/irssi.c

```diff
@@ -79,32 +80,20 @@ static int dirty, full_redraw;
 static GMainLoop *main_loop;
 int quitting;
 
-static const char *banner_text =
-	" ___           _\n"
-	"|_ _|_ _ _____(_)\n"
- 	" | || '_(_-<_-< |\n"
-	"|___|_| /__/__/_|\n"
-	"Irssi v" PACKAGE_VERSION " - http://www.irssi.org";
-
-static const char *firsttimer_text =
-	"- - - - - - - - - - - - - - - - - - - - - - - - - - - -\n"
-	"Hi there! If this is your first time using Irssi, you\n"
-	"might want to go to our website and read the startup\n"
-	"documentation to get you going.\n\n"
-	"Our community and staff are available to assist you or\n"
-	"to answer any questions you may have.\n\n"
-	"Use the /HELP command to get detailed information about\n"
-	"the available commands.\n"
-	"- - - - - - - - - - - - - - - - - - - - - - - - - - - -";
-
 static int display_firsttimer = FALSE;
+static int user_settings_changed = 0;
 
 
 static void sig_exit(void)
 {
         quitting = TRUE;
 }
 
+static void sig_settings_userinfo_changed(gpointer changedp)
+{
+	user_settings_changed = GPOINTER_TO_INT(changedp);
```

Comment:
Changing `=` to `|=` makes this mechanism more flexible in case we ever need to print other welcome warnings from outside `init_userinfo`.